frontend/src/app/main/ui/workspace/viewport/actions.cljs

```diff
@@ -284,8 +282,7 @@
 (defn on-key-up []
  (mf/use-callback
   (fn [event]
-    (let [key    (.-keyCode event)
-          key    (.normalizeKeyCode KeyCodes key)
+    (let [key    (.-key event)
```

Comment:
I don't think you need the `key` anymore.

Reply:
same thing